pyscriptjs/Makefile

```diff
@@ -32,7 +32,6 @@ build:
 	npm run build
 
 test:
-	@echo "Tests are coming :( this is a placeholder and it's meant to fail!"
 	$(conda_run) pytest -vv $(ARGS) tests/ --log-cli-level=warning
```

Comment:
This will fail because playwright is not being installed anymore, after the latest changes